dummydns.py

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env python3
 
-import socket, struct, sys
+import socket, struct, sys, signal
 
-default_address_dummydns = ["", 53]
+default_address_dummydns = ["127.0.0.1", 53]
+
+def dummydns_ctrl_c_handler(signum, frame):
```

Comment:
Is this required for windows or just preference? I can't imagine python not properly throwing a KeyboardInterrupt, honestly.

Reply:
I see. Alright, I'll add this too, then.